src/main/asciidoc/index.adoc

```diff
@@ -2,7 +2,16 @@
 Oliver Gierke; Thomas Darimont; Christoph Strobl; Mark Pollack; Thomas Risberg; Mark Paluch; Jay Bryant
 :revnumber: {version}
 :revdate: {localdate}
+:linkcss:
```

Comment:
So far, I've preferred to have these settings in the build setup, as that means it can be managed centrally and doesn't have to be repeated in all `index.adoc` for all Spring Data projects. Is there a reason to add them here?